calabash-cucumber/lib/calabash-cucumber/store/preferences.rb

```diff
@@ -40,10 +107,8 @@ def ensure_preferences_dir
       def defaults
         {
           :version => VERSION,
-          :usage_tracking =>
-          {
-
-          }
+          :usage_tracking => "system_info",
+          :user_id => SecureRandom.uuid
```

Comment:
Subsequent versions will probably allow the user to set this :user_id to an email address.